pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -826,7 +825,15 @@ func getCRICadvisorStats(infos map[string]cadvisorapiv2.ContainerInfo) map[strin
 		if !isPodManagedContainer(&info) {
 			continue
 		}
-		stats[path.Base(key)] = info
+		if len(info.Spec.Aliases) > 0 {
```

Comment:
I wonder maybe instead we need a special check for systemd cgroup driver like here instead:

https://github.com/kubernetes/kubernetes/blob/5be21c50c269fc1d28e0bd31ab9dcb572ae7fac5/pkg/kubelet/stats/cadvisor_stats_provider.go#L307-L314

It would be nice if can avoid using `info.Spec.Aliases` since as you mentioned behavior of containerd/crio/docker are different.

Reply:
Really cool. Thanks @bobbypage . 
And since there're two kind of format of cgroupPath for systemd cgroup driver. I'll test with this kind of special check.

Reply:
Hi @bobbypage , I just changed to code to extract id from systemd-style cgroup path. Please help review again.